Classes/Composer/ScriptHandler.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
This script appears to be a duplicate from the base-distribution package (minus unit tests) -- will it be removed from there?